README.md

```diff
@@ -118,6 +118,15 @@ When constructing filepaths, including while using [Quarto's Includes](https://q
 
 Refer to the [`tachyons-flexbox.qmd`](templates/tachyons-flexbox.qmd) template for an example.
 
+### Asset files
+> All video, stylesheet, and font files must go in `/assets`, and referenced in `index.qmd` in the YAML header under `resources` so that they are correctly rendered with the rest of the site by Quarto.
+
+e.g.: 
```

Comment:
Please also delete these lines. There are oodles of docs site config options that we don't document in the readme — let's not start documenting stuff folks don't need to interact with.